### ISSUE TITLE
电影资源库可以直接选择smb或者webdav目录加载吗？

### ISSUE THREAD

**Issue by lonuo** [jellyfin/jellyfin#12089]
电影资源库可以直接选择smb或者webdav目录加载吗？
因为有两个nas，想用一个nas专门存视频，一个nas专门挂路由，docker之类的。

**Comment by Bond-009** [comment]
> Can the movie resource library directly select smb or webdav directory to load?

No, mounting network shares is out of scope for Jellyfin itself and should be done by your operating system.
